presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java

```diff
@@ -670,6 +829,11 @@ protected T computeNext()
         };
     }
 
+    private String toCharValue(Object value)
```

Comment:
- make this static, change the parameter type to String and add length parameter (don't hard-code 10)
- the chars test needs refactoring similar to varchar; afterwards it may be simpler to inline this logic